Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/CoronaTest.kt

```diff
@@ -38,6 +38,15 @@ interface CoronaTest {
 
     val isResultAvailableNotificationSent: Boolean
 
+    // Is the digital green certificate supported by the point of care that issued the test
```

Comment:
Are we now using DCC or DGC?

Reply:
We should use DCC as it is the official EU name, I will refactor in another PR. If you find it anywhere else, feel free to change it too :smile: